istioctl/cmd/loghelp.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2020 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+var (
+	helpFlags = map[string]bool{
+		"log_as_json":          true,
+		"log_stacktrace_level": true,
+		"log_target":           true,
+		"log_caller":           true,
+		"log_output_level":     true,
+		// istioctl also inherits support for log_rotate, log_rotate_max_age, log_rotate_max_backups,
+		// log_rotate_max_size, but these are rarely appropriate for a user-facing CLI so we ignore them
+	}
+)
+
+func logHelpCommand(rootCmd *cobra.Command) *cobra.Command {
+	return &cobra.Command{
+		Use:   "log-help",
```

Comment:
Is it worth making help its own subcommand? `istioctl help log`?

Reply:
I believe `log-help` was a suggestion from @therealmitchconnors .  My intention was to keep the ginormus `--log_stacktrace_level` info off the help that prints by default.

Reply:
My concern was more about increasing the number of top-level commands.

Reply:
@liamawhite There is a `kubectl options` command, which lists all the rare options of kubectl.  Would you prefer it if we change this to `istioctl options` so that we can dump other rare options there?

Reply:
Yes, that's better than my suggestion 🙂

Reply:
@liamawhite OK, done, please re-review.